doc/ref_arch/kubernetes/chapters/chapter06.md

```diff
@@ -10,9 +10,46 @@
 
 <a name="6.1"></a>
 ## 6.1 Introduction
+The adoption of cloud-native allows development teams to move fast, deploy software efficiently, and operate at an unprecedented scale. 
+Cloud-native applications offer the benefits by providing applications as small microservices (either containers or serverless), and managing them through DevOps processes. To build these applications, organisations need a cloud-native infrastructure which provides flexibility and enough services for developers. Cloud-native infrastructures should provide a platform for the build and release team to effectively perform continuous delivery of its applications, with close to zero human intervention. With Cloud-native, the industy requires a radically different apporach to security in contrast to monolithic architecture. 
```

Comment:
"To build these applications, organisations need a **cloud-native infrastructure which provides flexibility and enough services for developers**. Cloud-native infrastructures should provide a **platform for the build and release team to effectively perform continuous delivery** of its applications, with close to zero human intervention." 

1. Why are requirements for a "cloud native infrastructure" part of a "security" chapter?
2. What would constitute "enough" services for developers?

"With Cloud-native, the industy requires a radically different apporach to security in contrast to monolithic architecture."  
Should we be comparing the requirements against "monolithic architecture" or against virtualised environments as in RA-1 Chapter 5 and/or against the security guidelines in RM Chapter 07?